lib/awful/client.lua

```diff
@@ -727,11 +727,10 @@ end
 -- fullscreen, maximized, or horizontally maximized?
 --
 -- This property is read only.
--- @property immobilized
+-- @property is_immobilized_horizontal
```

Comment:
Good catch, but I think the property are called `immobilized_horizontal` and `immobilized_vertical`. The `is_` is removed when the getter becomes a property (at least it should be).

Ref: https://github.com/awesomeWM/awesome/blob/master/lib/gears/object/properties.lua#L40